bugbug/model.py

```diff
@@ -455,6 +456,19 @@ def train(self, importance_cutoff=0.15, limit=None):
         y_pred = self.clf.predict(X_test)
         y_pred = self.le.inverse_transform(y_pred)
 
+        # calibrating the model
```

Comment:
We should have the calibrating process as optional. It could be enabled per model after checking the performance with and without enabling it and making sure that it improves the performance.

Reply:
I have no idea how to do that :sweat_smile: . Can you please guide me to do so.

Reply:
We can compare the Mean Square error of both without and with Calibration and then choose the better performing model, how bout this?

Reply:
>  have no idea how to do that 😅 . Can you please guide me to do so.

We could do something similar to the cross validation:

https://github.com/mozilla/bugbug/blob/f9906057a5281b8913fb5a92edbe73440953581b/bugbug/model.py#L368-L387